.github/workflows/publish.yml

```diff
@@ -21,7 +21,19 @@ jobs:
         run: npm install
       - name: NPM run build
         run: npm run build
-      - name: Publish to NPM
-        run: npm publish --access public
+      - name: Get npm tag name if needed
```

Comment:
Also this step seems to be executed anyway, should we remove "if needed" in its name? And add it to the next step "Publish to NPM latest if needed(possible/necessary/available)"

Reply:
so for semver they call alpha/beta/etc to pre-release. We so far go with beta but i keep the script which is just to extract the pre-release part to be generic in case we need to change name later. In this case we tag if if it is a pre-release. The tag name is the prelease name.